codes.md

```diff
@@ -280,8 +293,10 @@ spoke about the drawing and their mental model
       - [Session 024](https://osf.io/d8pke/) (second drawing, but also
         discussed as abstracted from original)
     - Bar chart:
-      - [Session 023](https://osf.io/prtw4/)
-      - [Session 032](https://osf.io/y8z4r/)
+      - Per station:
+        - [Session 023](https://osf.io/prtw4/)
+      - Per station, grouped by building type:
+        - [Session 032](https://osf.io/y8z4r/) (drawn)
```

Comment:
032 is grouped per station, but stations may have more than 1 bar (per building type)

Reply:
Will clarify as "faceted by building type"